### PR TITLE
All the CLI tests are passing

### PR DESCRIPTION
**Pull request by Aylr**
## What's Here

* continued progress giving `DataContext` more ability to determine the project's init state pulling out complexity from the cli into easily tested areas.
* really a whole lot more
* all CLI tests are passing on python3

### CODE REVIEW

File: great_expectations/cli/datasource.py [hunk 1]
```diff
@@ -272,21 +272,11 @@ def add_datasource(context, choose_one_data_asset=False):
...
         datasource_name = "files_datasource"
-
-        # configuration = PandasDatasource.build_configuration(generators={
-        #     "default": {
-        #         "class_name": "PassthroughGenerator",
-        #     }
-        # }
-        # )
-
-
```
Comment (Aylr) [inline great_expectations/cli/datasource.py:283]: bye bye

File: great_expectations/data_context/data_context.py [hunk 2]
```diff
@@ -1507,13 +1512,55 @@ def does_config_exist_on_disk(cls, context_root_dir):
 
     @classmethod
     def is_project_initialized(cls, ge_dir):
```
Comment (eugmandel) [inline great_expectations/data_context/data_context.py:1514]: "initialized" sounds confusing for me - my guess would be that to be initialized a context must have the config file and the basic directory.  Maybe "activated"?

**Review by eugmandel — APPROVED** [review]
LGTM!

File: great_expectations/cli/datasource.py [hunk 3]
```diff
@@ -461,7 +451,7 @@ def _add_sqlalchemy_datasource(context, prompt_for_datasource_name=True):
 Since we could not connect to the database, you can complete troubleshooting in the configuration files documented here:
 <blue>https://docs.greatexpectations.io/en/latest/tutorials/add-sqlalchemy-datasource.html?utm_source=cli&utm_medium=init&utm_campaign={3:s}#{4:s}</blue> .
 
-After you connect to the datasource, run great_expectations datasource profile to continue.
+After you connect to the datasource, run great_expectations init to continue.
```
Comment (jcampbell) [inline great_expectations/cli/datasource.py:454]: Quick confirmation...the new flow is to re-run init and not have init stop at the "your project is complete" check, right?

File: great_expectations/cli/init_messages.py [hunk 4]
```diff
@@ -55,8 +55,9 @@
 OK. <green>Slack is set up.</green> To modify this in the future please see the slack section in the CLI init getting started guide."""
 
 ONBOARDING_COMPLETE = """
-Done. You may see new files in `<yellow>great_expectations/uncommitted</yellow>`.
-  - Now add secrets to <yellow>great_expectations/uncommitted/config_variables.yml</yellow> to finish onboarding.
+Great Expectations added some missing files required to run.
+  - You may see new files in `<yellow>great_expectations/uncommitted</yellow>`.
+  - You may need to add secrets to <yellow>great_expectations/uncommitted/config_variables.yml</yellow> to finish onboarding.
```
Comment (jcampbell) [inline great_expectations/cli/init_messages.py:60]: <3 <3 <3

**Review by jcampbell — APPROVED** [review]

**Pull request merged**